qiskit/scheduler/methods/basic.py

```diff
@@ -75,7 +75,7 @@ def update_times(inst_qubits: List[int], time: int = 0) -> None:
 
     timed_schedules = [(time, cpd.schedule) for time, cpd in zip(start_times, circ_pulse_defs)
                        if not isinstance(cpd.schedule, Barrier)]
-    return Schedule(*timed_schedules, name=circuit.name)
+    return Schedule(*timed_schedules, name=circuit.name, inplace=False)
```

Comment:
@taalexander do you think False makes sense here? Or should I leave it as default (None), with single warning?